guides/source/active_record_migrations.md

```diff
@@ -5,35 +5,44 @@ Active Record Migrations
 
 Migrations are a feature of Active Record that allows you to evolve your
 database schema over time. Rather than write schema modifications in pure SQL,
-migrations allow you to use a Ruby DSL to describe changes to your tables.
+migrations allow you to use a Ruby Domain Specific Language (DSL) to describe
+changes to your tables.
 
 After reading this guide, you will know:
 
-* The generators you can use to create them.
+* The generators you can use to create migrations.
 * The methods Active Record provides to manipulate your database.
-* The rails commands that manipulate migrations and your schema.
+* The details of manipulating migrations and your schema.
+* Changing existing migrations.
 * How migrations relate to `schema.rb`.
+* Referential integrity.
 
 --------------------------------------------------------------------------------
 
 Migration Overview
 ------------------
 
-Migrations are a convenient way to
-[alter your database schema over time](https://en.wikipedia.org/wiki/Schema_migration)
-in a consistent way. They use a Ruby DSL so that you don't have to
-write SQL by hand, allowing your schema and changes to be database independent.
+Migrations are a convenient way to [alter your database schema over
+time](https://en.wikipedia.org/wiki/Schema_migration) in a consistent way. They
+use a Ruby [DSL](https://en.wikipedia.org/wiki/Domain-specific_language) so that
+you don't have to write SQL by hand, allowing your schema and changes to be
+database independent. We recommend that you read the guides for [Active Record
+Basics](active_record_basics.html) and the [Active Record
+Associations](association_basics.html) to learn about the some of the concepts
+mentioned here.
 
 You can think of each migration as being a new 'version' of the database. A
 schema starts off with nothing in it, and each migration modifies it to add or
 remove tables, columns, or entries. Active Record knows how to update your
```

Comment:
I suspect what was initially meant by "entries" is "making changes to existing data (i.e. entries)", and while migrations primarily focus on modifying the database schema rather than directly altering existing data, you can technically include operations that manipulate data as part of the schema changes (but from what I understand, it's not a best practice). 

Hence, I'm feeling a bit at odds on whether to change this to "A schema starts off with nothing in it, and each migration modifies it to add or remove tables, columns, or by making changes to existing data" or omit it completely, but if we omit it then we're ignoring the fact that we can manipulate data entries as part of the schema.